integrations/clickup/pyproject.toml

```diff
@@ -0,0 +1,102 @@
+[tool.poetry]
+name = "clickup"
+version = "0.1.0"
+description = "Integration module for ClickUp"
+authors = ["Iyanuoluwa Adebayo <ioluwadunsinadebayo@gmail.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+port_ocean = { version = "^0.9.1", extras = ["cli"] }
```

Comment:
please upgrade your ocean version to the latest (0.9.4)

Reply:
This has been done